Snowflake.Data/Core/Session/SFSessionProperty.cs

```diff
@@ -249,7 +250,7 @@ internal static SFSessionProperties ParseConnectionString(string connectionStrin
 
             if (password != null && password.Length > 0)
             {
-                properties[SFSessionProperty.PASSWORD] = new NetworkCredential(string.Empty, password).Password;
+                properties[SFSessionProperty.PASSWORD] = SecureStringHelper.Decode(password);

```

Comment:
what worries me that for a security reason this was held encrypted (within SecureString) and now we're taking that option down

Reply:
this PR we does not change anything. This change is only to replace `new NetworkCredential(string.Empty, password).Password` by function execection with exactly the same code.